.circleci/config.yml

```diff
@@ -138,6 +167,7 @@ jobs:
           -X POST https://slack.com/api/chat.postMessage
 
     - post_steps
+    - release2pypi
```

Comment:
As discussed, I've now created a separate job for the release to PyPI. Subsequently added two extra jobs to the workflow. One for PR's and one for the release, as I thought that in case of a merge, the release should be required.